contrib/depends/packages/unwind.mk

```diff
@@ -1,12 +1,10 @@
 package=unwind
-$(package)_version=1.2
+$(package)_version=1.5.0
```

Comment:
Is this patch your original? What does "/libunwind#7249" refer to? Libunwind doesn't have that PR number, and Monero's PR 7249 is on something completely different.

If this patch is your original and is also applicable to Monero, please send a PR to Monero first and refer to it here.
https://github.com/aeonix/aeon/blob/master/CONTRIBUTING.md#proposing-changes

Reply:
My bad, that is for https://github.com/libunwind/libunwind/pull/166 . Recommendation is to update to 1.5.0 . I will look into testing monero.